### PR TITLE
feat: account profile - rating

### PR DESCRIPTION
**Pull request by stanleyyconsensys**
This PR is adding a component for account profile review rating

it render star by the given parameter 'rate' and 'maxRate'

where rate is refer to the current rating star, and maxRate is refer to maximum rating, 
 
`Note: Star design and colour is not confirmed on dark theme`

image

### CODE REVIEW

**Review by ritave — APPROVED** [review]

File: src/components/Rating.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  const star: ReactElement[] = useMemo(() => {
+    const _star: ReactElement[] = [];
+    for (let i = 0; i < maxRate; i++) {
+      _star.push(
+        i < rate ? (
+          <StarFilledIcon
+            key={i}
+            data-testid={`filled-${i}`}
+            fill={colorMode === 'light' ? '#6A737D' : '#BBC0C5'}
```
Comment (Mrtenz) [inline src/components/Rating.tsx:26]: Use a theme value please.

File: src/components/Rating.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          />
+        ) : (
+          <StarIcon
+            key={i}
+            data-testid={`empty-${i}`}
+            fill={colorMode === 'light' ? '#BBC0C5' : '#6A737D'}
```
Comment (Mrtenz) [inline src/components/Rating.tsx:32]: Use a theme value please.

File: src/components/Rating.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+export const Rating: FunctionComponent<RatingProps> = ({
+  rate,
+  maxRate = 5,
+}) => {
+  const { colorMode } = useColorMode();
+
+  const star: ReactElement[] = useMemo(() => {
+    const _star: ReactElement[] = [];
```
Comment (Mrtenz) [inline src/components/Rating.tsx:19]:
```suggestion
    const stars: ReactElement[] = [];
```

File: src/components/icons/index.ts [hunk 4]
```diff
@@ -79,4 +81,11 @@ export const TransactionInsightsIcon = wrapIcon(
   defineMessage`Security`,
 );
 
+export const StarFilledIcon = wrapIcon(
+  BaseStarFilledIcon,
+  defineMessage`MetaMask`,
+);
+
+export const StarIcon = wrapIcon(BaseStarIcon, defineMessage`MetaMask`);
```
Comment (Mrtenz) [inline src/components/icons/index.ts:89]: These messages seem wrong.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Comment by alainncls** [comment]
Closed in favour of #41

**Pull request closed without merging**